src/sentry/issues/grouptype.py

```diff
@@ -159,6 +161,12 @@ def allow_post_process_group(cls, organization: Organization) -> bool:
 
         return features.has(cls.build_post_process_group_feature_name(), organization)
 
+    @classmethod
+    def should_generate_escalating_forecasts(cls, organization: Organization) -> bool:
```

Comment:
Looking at this again, we're using this field for more than just the forecasts. Maybe `should_detect_escalation` captures both the detection and the forecasting?